### ISSUE TITLE
Error while helm install jobs-app

### ISSUE THREAD

**Issue by east4ming** [isovalent/cilium-grafana-observability-demo#8]
After running: 
```bash
helm dep build ./helm/jobs-app
helm upgrade jobs-app ./helm/jobs-app \
  --install \
  --wait \
  --create-namespace \
  --namespace tenant-jobs \
  -f helm/jobs-app-values.yaml
```

helm install failed, coreapi/loader/resumes CrashLoopBackoff, logs:

coreapi pod:
```logs
[2023-05-03 09:53:37,206] INFO in es: Connecting to elasticsearch http://elastic:changeme@elasticsearch-master.tenant-jobs.svc.cluster.local:9200/                                                                                                                                                                    │
│ Traceback (most recent call last):                                                                                                                                                                                                                                                                                    │
│   File "/usr/local/bin/gunicorn", line 8, in <module>                                                                                                                                                                                                                                                                 │
│     sys.exit(run())                                                                                                                                                                                                                                                                                                   │
│   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 67, in run                                                                                                                                                                                                                              │
│     WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()                                                                                                                                                                                                                                                          │
│   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 231, in run                                                                                                                                                                                                                                │
│     super().run()                                                                                                                                                                                                                                                                                                     │
│   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 72, in run                                                                                                                                                                                                                                 │
│     Arbiter(self).run()                                                                                                                                                                                                                                                                                               │
│   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 58, in __init__                                                                                                                                                                                                                             │
│     self.setup(app)                                                                                                                                                                                                                                                                                                   │
│   File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 118, in setup                                                                                                                                                                                                                               │
│     self.app.wsgi()                                                                                                                                                                                                                                                                                                   │
│   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi                                                                                                                                                                                                                                │
│     self.callable = self.load()                                                                                                                                                                                                                                                                                       │
│   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load                                                                                                                                                                                                                             │
│     return self.load_wsgiapp()                                                                                                                                                                                                                                                                                        │
│   File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp                                                                                                                                                                                                                     │
│     return util.import_app(self.app_uri)                                                                                                                                                                                                                                                                              │
│   File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 412, in import_app                                                                                                                                                                                                                             │
│     app = app(*args, **kwargs)                                                                                                                                                                                                                                                                                        │
│   File "/opt/app/app.py", line 120, in gunicorn_start                                                                                                                                                                                                                                                                 │
│     setup(app)                                                                                                                                                                                                                                                                                                        │
│   File "/opt/app/bootstrap.py", line 11, in setup                                                                                                                                                                                                                                                                     │
│     bootstrap(app, es, api)                                                                                                                                                                                                                                                                                           │
│   File "/opt/app/bootstrap.py", line 17, in bootstrap                                                                                                                                                                                                                                                                 │
│     es.indices.delete(index='analytics', ignore=[400, 404])                                                                                                                                                                                                                                                           │
│   File "/usr/local/lib/python3.9/site-packages/elasticsearch/_sync/client/utils.py", line 414, in wrapped                                                                                                                                                                                                             │
│     return api(*args, **kwargs)                                                                                                                                                                                                                                                                                       │
│   File "/usr/local/lib/python3.9/site-packages/elasticsearch/_sync/client/indices.py", line 690, in delete                                                                                                                                                                                                            │
│     return self.perform_request(  # type: ignore[return-value]                                                                                                                                                                                                                                                        │
│   File "/usr/local/lib/python3.9/site-packages/elasticsearch/_sync/client/_base.py", line 390, in perform_request                                                                                                                                                                                                     │
│     return self._client.perform_request(                                                                                                                                                                                                                                                                              │
│   File "/usr/local/lib/python3.9/site-packages/elasticsearch/_sync/client/_base.py", line 286, in perform_request                                                                                                                                                                                                     │
│     meta, resp_body = self.transport.perform_request(                                                                                                                                                                                                                                                                 │
│   File "/usr/local/lib/python3.9/site-packages/elastic_transport/_transport.py", line 329, in perform_request                                                                                                                                                                                                         │
│     meta, raw_data = node.perform_request(                                                                                                                                                                                                                                                                            │
│   File "/usr/local/lib/python3.9/site-packages/elastic_transport/_node/_http_urllib3.py", line 199, in perform_request                                                                                                                                                                                                │
│     raise err from None                                                                                                                                                                                                                                                                                               │
│ elastic_transport.ConnectionError: Connection error caused by: ConnectionError(Connection error caused by: ProtocolError(('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))))  
```

loader:

```log
 {"level":"ERROR","timestamp":"2023-05-03T09:54:10.430Z","logger":"kafkajs","message":"[BrokerPool] Closed connection","retryCount":4,"retryTime":3768}                                                                                                                                                                │
│ {"level":"ERROR","timestamp":"2023-05-03T09:54:15.304Z","logger":"kafkajs","message":"[BrokerPool] Closed connection","retryCount":5,"retryTime":6436}                                                                                                                                                                │
│ {"level":"error","message":"[resume/producer] Closed connection Closed connection","name":"KafkaJSNumberOfRetriesExceeded","retriable":false,"retryCount":5,"retryTime":6436,"stack":"KafkaJSNonRetriableError\n  Caused by: KafkaJSConnectionClosedError: Closed connection\n    at Socket.onEnd (/opt/app/node_modu │
│ Stream closed EOF for tenant-jobs/loader-5656b9859-4mpqx (loader) 
```

resume:

```log
 {"level":"info","message":"[resume/consumer] resumes worker started"}                                                                                                                                                                                                                                                 │
│ {"level":"ERROR","timestamp":"2023-05-03T09:59:38.145Z","logger":"kafkajs","message":"[BrokerPool] Closed connection","retryCount":0,"retryTime":306}                                                                                                                                                                 │
│ {"level":"ERROR","timestamp":"2023-05-03T09:59:39.452Z","logger":"kafkajs","message":"[Connection] Connection timeout","broker":"jobs-app-kafka-brokers.tenant-jobs.svc.cluster.local:9092","clientId":"resumes-consumer"}                                                                                            │
│ {"level":"ERROR","timestamp":"2023-05-03T09:59:39.453Z","logger":"kafkajs","message":"[BrokerPool] Failed to connect to seed broker, trying another broker from the list: Connection timeout","retryCount":1,"retryTime":668}                                                                                         │
│ {"level":"ERROR","timestamp":"2023-05-03T09:59:45.131Z","logger":"kafkajs","message":"[Connection] Connection error: read ECONNRESET","broker":"jobs-app-kafka-brokers.tenant-jobs.svc.cluster.local:9092","clientId":"resumes-consumer","stack":"Error: read ECONNRESET\n    at TCP.onStreamRead (node:internal/stre │
│ {"level":"ERROR","timestamp":"2023-05-03T09:59:45.132Z","logger":"kafkajs","message":"[BrokerPool] Failed to connect to seed broker, trying another broker from the list: Connection error: read ECONNRESET","retryCount":2,"retryTime":1394}  
```

**Comment by vitality411** [comment]
I had the same issue when running on K8s 1.25. There was no Kafka or Zookeeper as strimizi-cluster-operator was unable to create v1beta PodDisruptionBudgets. Support for v1 PodDisruptionBudgets was added in strimzi-kafka-operator 0.29.0 (see [#6066](https://github.com/strimzi/strimzi-kafka-operator/issues/6066) and [#6313](https://github.com/strimzi/strimzi-kafka-operator/pull/6313)). So I changed the version in helm/jobs-app/Chart.yaml and redeployed. After Kafka and Zookeeper have been deployed, the other pods also were running.
